.bazelrc

```diff
@@ -1,3 +1 @@
-common       --config=rules
-common:v7.2 --config=rules_v7.2
```

Comment:
Do you know why this existed too?

Why not do `foo:v7.2 bar` instead of `foo:rules_v7.2 bar`?